src/newViews/AnalysisModule/Analysis/AnalysisPillar/index.tsx

```diff
@@ -28,6 +29,19 @@ import { calcPercent } from '#utils/safeCommon';
 import _ts from '#ts';
 import styles from './styles.scss';
 
+function shouldHideEdit({
+    hasAnalysisFramework,
+    entryPermissions,
+}: {
+    hasAnalysisFramework: boolean;
+    entryPermissions: {
+        create: boolean;
+        modify: boolean;
+    };
+}) {
+    return (!hasAnalysisFramework || !(entryPermissions.create || entryPermissions.modify));
```

Comment:
Why not just use modify?

Reply:
We've duplicated the edit entries permissions here